processor/src/decorators/mod.rs

```diff
@@ -189,15 +189,15 @@ where
         let elm_arr = [inv_elm];
         let coeffs = Ext2Element::as_base_elements(&elm_arr);
 
-        self.advice_provider.write_tape(AdviceSource::Value(coeffs[1]))?;
-        self.advice_provider.write_tape(AdviceSource::Value(coeffs[0]))?;
+        self.advice_provider.write_stack(AdviceSource::Value(coeffs[1]))?;
+        self.advice_provider.write_stack(AdviceSource::Value(coeffs[0]))?;
 
         Ok(())
     }
 
     /// Given evaluations of a polynomial over some specified domain, this routine
     /// interpolates the evaluations into a polynomial in coefficient form and writes
-    /// the result into the advice tape. The interpolation is performed using the iNTT
+    /// the result into the advice stack. The interpolation is performed using the iNTT
```

Comment:
Let's update: "writes the results into the advice stack" to "pushes the results onto the advice stack".

Reply:
Done!